pybloqs/static/__init__.py

```diff
@@ -1,78 +1,80 @@
-import itertools
-
-from pybloqs.html import js_elem, append_to, root, render
-from pybloqs.util import encode_string
-
 from pkg_resources import resource_filename
-
-
 from six import StringIO
 
+from pybloqs.html import js_elem, css_elem,  render
+from pybloqs.util import encode_string
+import os
+from io import open
+
 
 class Resource(object):
-    def __init__(self, name, extension):
+    def __init__(self, file_name=None, extension='', content_string=None, name=None):
         """
         An external script dependency definition.
 
         :param name: Canonical name of the script.
-        :param extension: Script file extension.
         """
-        self._name = name
-        self._extension = extension
-        self._hash = hash(name)
+        # Do XOR for checking if one of the two parameters is set
```

Comment:
I find this comment a bit confusing as the following expression actually checks three parameters.

Maybe:
```suggestion
        # Do XOR for checking if either file_name, or content_string and name are set 
```